src/App.svelte

```diff
@@ -20,8 +17,8 @@
 
 <div>
```

Comment:
Ja nogmaals kan ook een svelte ding zijn maar ik zie hier vooral een div die geen styling of purpose heeft